README.md

```diff
@@ -30,10 +30,8 @@ example below.
 import { authenticationMiddleware } from "@kiwicom/iam";
 
 const options = {
-  // These two fields are mandatory and are used to check what service
-  // is the token intended for (audience).
-  iapProjectNumber: process.env.IAP_PROJECT_NUMBER,
```

Comment:
This should not be dropped immediately, I would add console warning that this is deprecated.